unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/BoxShape/ECSBoxShapeComponentHandler.cs

```diff
@@ -57,6 +57,8 @@ internal void DisposeMesh(IParcelScene scene)
                 ECSComponentsUtils.DisposeMeshInfo(meshesInfo);
             if(rendereable != null)
                 ECSComponentsUtils.RemoveRendereableFromDataStore( scene.sceneData.id,rendereable);
+            if(model != null)
+                DataStore.i.ecs7.pendingSceneResources.DecreaseRefCount((scene.sceneData.id, model));
```

Comment:
If a component fails I m not sure if we should load the scene anyway. 

I will add it because right now the initial loading is waiting for other scenes to load, but if a component of the scene fail loading we should handle that appart

Reply:
If component fail, we should load the scene anyway.
Handling component failing to load should be responsibility of other system